chromadb/__init__.py

```diff
@@ -22,6 +22,55 @@ def get_settings() -> Settings:
     return __settings
 
 
+def EphemeralClient(settings: Settings = Settings()) -> API:
+    """
+    Creates an in-memory instance of Chroma. This is useful for testing and
+    development, but not recommended for production use.
+    """
+    settings.chroma_db_impl = "chromadb.db.duckdb.DuckDB"
+
+    return Client(settings)
+
+
+def PersistentClient(path: str = "./chroma", settings: Settings = Settings()) -> API:
```

Comment:
Hmmm, are we going to get into weird situations where people try to override the defaults thinking it will do something? I wonder if we want to error or if its rare enough to ignore

Reply:
The issue is that init of the `Settings()` objects brings with it default values... so there will *always* be conflicts. This makes it hard to "detect" what fields the user is expressly settings vs are just default values and hard to know when to fire a warning.